core/block/editor.go

```diff
@@ -33,17 +33,20 @@ var ErrOptionUsedByOtherObjects = fmt.Errorf("option is used by other objects")
 
 type FileUploadRequest struct {
 	pb.RpcFileUploadRequest
-	Origin model.ObjectOrigin
+	Origin     model.ObjectOrigin
```

Comment:
Looks like Origin and ImportType always go together. Does it make sense to encapsulate it into domain type? Like `domain.ObjectOrigin`.

Constructors will be like:
```
domain.ObjectOriginImport(importType)
domain.ObjectOriginNone()
...
domain.ObjectOriginBookmark()
```

Reply:
Fixed